megamek/src/megamek/common/verifier/TestAero.java

```diff
@@ -603,12 +603,17 @@ public double getWeightPowerAmp() {
                 if ((linkedBy != null) && 
                         (linkedBy.getType() instanceof MiscType) && 
                         linkedBy.getType().hasFlag(MiscType.F_PPC_CAPACITOR)){
-                    weight += ((MiscType)linkedBy.getType()).getTonnage(aero);
+                    weight += linkedBy.getType().getTonnage(aero);
                 }
             }
-            // Power amp weighs: 
+            for (Mounted m : aero.getMisc()) {
+                if (m.getType().hasFlag(MiscType.F_CLUB) && m.getType().hasSubType(MiscType.S_SPOT_WELDER)) {
```

Comment:
Aerospace fighters can mount spot welders? wtf?

Reply:
and clubs.

Reply:
Well, never mind, just spot welders, which are apparently a subset of club.

Reply:
I'm sorry, what?

Is that meant to only be for LAMs?

Reply:
I guess that's a theoretically legit use case. Well, whatever, we're just implementing the rules.

Reply:
That's for conventional fighters, and I was just blindly thinking about how they use the same rules as vehicles for determining power amplifier weight without any consideration for the equipmrnt in question. Conventional fighter cannot in fact mount spot welders so while this doesn't hurt anything, the check is meaningless.